cmd/fabric8-jenkins-proxy/main.go

```diff
@@ -227,3 +214,25 @@ func setupSignalChannel(cancel context.CancelFunc) {
 		cancel()
 	}()
 }
+
+func getAPIServer(api api.ProxyAPI) *http.Server {
```

Comment:
nitpick: Golang getters just use the `Noun` instead of `GetNoun` see: https://golang.org/doc/effective_go.html#Getters

I don't think this is a getter since calling it creates a new  ApiServer all the time, hence in go-speak I would call it NewApiServer()